127.0.0.1 - - [02/Apr/2025 19:30:10] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:30:14] "GET /dub.html HTTP/1.1" 304 -
127.0.0.1 - - [02/Apr/2025 19:30:14] "GET /assets/js/dubbing-service.js HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:30:14] code 404, message File not found
127.0.0.1 - - [02/Apr/2025 19:30:14] "GET /assets/js/libs/axios.min.js.map HTTP/1.1" 404 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-group.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-volume.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-pitch.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-speed.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-multi.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-effect.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/js/common.js HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/js/auth.js HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/js/libs/axios.min.js HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /ai-script-generator.js HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/js/text-area-fix.js HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-music.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/images/avatar-xiaoyu.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-redo.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-undo.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/images/avatar-6.png HTTP/1.1" 200 -
----------------------------------------
Exception occurred during processing of request from ('127.0.0.1', 50380)
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-play.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/icons/icon-filter.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/images/avatar-3.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/images/avatar-4.png HTTP/1.1" 200 -
----------------------------------------
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/images/avatar-5.png HTTP/1.1" 200 -
Exception occurred during processing of request from ('127.0.0.1', 50422)
----------------------------------------
----------------------------------------
----------------------------------------
Exception occurred during processing of request from ('127.0.0.1', 50434)
Exception occurred during processing of request from ('127.0.0.1', 50440)
Exception occurred during processing of request from ('127.0.0.1', 50432)
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 692, in process_request_thread
    self.finish_request(request, client_address)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 692, in process_request_thread
    self.finish_request(request, client_address)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 1311, in finish_request
    self.RequestHandlerClass(request, client_address, self,
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 692, in process_request_thread
    self.finish_request(request, client_address)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 1311, in finish_request
    self.RequestHandlerClass(request, client_address, self,
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 692, in process_request_thread
    self.finish_request(request, client_address)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 1311, in finish_request
    self.RequestHandlerClass(request, client_address, self,
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 692, in process_request_thread
    self.finish_request(request, client_address)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 672, in __init__
    super().__init__(*args, **kwargs)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 672, in __init__
    super().__init__(*args, **kwargs)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 1311, in finish_request
    self.RequestHandlerClass(request, client_address, self,
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 672, in __init__
    super().__init__(*args, **kwargs)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 761, in __init__
    self.handle()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 672, in __init__
    super().__init__(*args, **kwargs)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 1311, in finish_request
    self.RequestHandlerClass(request, client_address, self,
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 761, in __init__
    self.handle()
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 761, in __init__
    self.handle()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 436, in handle
    self.handle_one_request()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 436, in handle
    self.handle_one_request()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 672, in __init__
    super().__init__(*args, **kwargs)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 436, in handle
    self.handle_one_request()
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 761, in __init__
    self.handle()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 436, in handle
    self.handle_one_request()
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 761, in __init__
    self.handle()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 436, in handle
    self.handle_one_request()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 424, in handle_one_request
    method()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 424, in handle_one_request
    method()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 679, in do_GET
    self.copyfile(f, self.wfile)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 424, in handle_one_request
    method()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 424, in handle_one_request
    method()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 878, in copyfile
    shutil.copyfileobj(source, outputfile)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 679, in do_GET
    self.copyfile(f, self.wfile)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 679, in do_GET
    self.copyfile(f, self.wfile)
  File "/home/hoey/miniconda3/lib/python3.12/shutil.py", line 204, in copyfileobj
    fdst_write(buf)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 878, in copyfile
    shutil.copyfileobj(source, outputfile)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 679, in do_GET
    self.copyfile(f, self.wfile)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 878, in copyfile
    shutil.copyfileobj(source, outputfile)
  File "/home/hoey/miniconda3/lib/python3.12/shutil.py", line 204, in copyfileobj
    fdst_write(buf)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 878, in copyfile
    shutil.copyfileobj(source, outputfile)
  File "/home/hoey/miniconda3/lib/python3.12/shutil.py", line 204, in copyfileobj
    fdst_write(buf)
  File "/home/hoey/miniconda3/lib/python3.12/shutil.py", line 204, in copyfileobj
    fdst_write(buf)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 840, in write
    self._sock.sendall(b)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 840, in write
    self._sock.sendall(b)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 840, in write
    self._sock.sendall(b)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 424, in handle_one_request
    method()
BrokenPipeError: [Errno 32] Broken pipe
BrokenPipeError: [Errno 32] Broken pipe
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 679, in do_GET
    self.copyfile(f, self.wfile)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 840, in write
    self._sock.sendall(b)
----------------------------------------
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 878, in copyfile
    shutil.copyfileobj(source, outputfile)
BrokenPipeError: [Errno 32] Broken pipe
  File "/home/hoey/miniconda3/lib/python3.12/shutil.py", line 204, in copyfileobj
    fdst_write(buf)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 840, in write
    self._sock.sendall(b)
----------------------------------------
BrokenPipeError: [Errno 32] Broken pipe
----------------------------------------
----------------------------------------
BrokenPipeError: [Errno 32] Broken pipe
----------------------------------------
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /favicon.ico HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/images/avatar-1.png HTTP/1.1" 200 -
127.0.0.1 - - [02/Apr/2025 19:34:04] "GET /assets/images/avatar-2.png HTTP/1.1" 200 -
----------------------------------------
Exception occurred during processing of request from ('127.0.0.1', 50454)
----------------------------------------
Exception occurred during processing of request from ('127.0.0.1', 50456)
Traceback (most recent call last):
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 692, in process_request_thread
    self.finish_request(request, client_address)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 1311, in finish_request
    self.RequestHandlerClass(request, client_address, self,
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 672, in __init__
    super().__init__(*args, **kwargs)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 761, in __init__
    self.handle()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 436, in handle
    self.handle_one_request()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 424, in handle_one_request
    method()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 679, in do_GET
    self.copyfile(f, self.wfile)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 878, in copyfile
    shutil.copyfileobj(source, outputfile)
  File "/home/hoey/miniconda3/lib/python3.12/shutil.py", line 204, in copyfileobj
    fdst_write(buf)
Traceback (most recent call last):
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 840, in write
    self._sock.sendall(b)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 692, in process_request_thread
    self.finish_request(request, client_address)
BrokenPipeError: [Errno 32] Broken pipe
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 1311, in finish_request
    self.RequestHandlerClass(request, client_address, self,
----------------------------------------
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 672, in __init__
    super().__init__(*args, **kwargs)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 761, in __init__
    self.handle()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 436, in handle
    self.handle_one_request()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 424, in handle_one_request
    method()
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 679, in do_GET
    self.copyfile(f, self.wfile)
  File "/home/hoey/miniconda3/lib/python3.12/http/server.py", line 878, in copyfile
    shutil.copyfileobj(source, outputfile)
  File "/home/hoey/miniconda3/lib/python3.12/shutil.py", line 204, in copyfileobj
    fdst_write(buf)
  File "/home/hoey/miniconda3/lib/python3.12/socketserver.py", line 840, in write
    self._sock.sendall(b)
BrokenPipeError: [Errno 32] Broken pipe
----------------------------------------
